lib/addons/topics-api.ts

```diff
@@ -0,0 +1,32 @@
+import OptableSDK from "../sdk";
+
+declare module "../sdk" {
+    export interface OptableSDK {
+        getTopics: () => void;
+    }
+}
+
+declare global {
+    interface Document {
+        browsingTopics?: () => Promise<Array<{
```

Comment:
Should we put a comment with a link to https://patcg-individual-drafts.github.io/topics/#dictdef-browsingtopic
for the official specs of this returned object?